lookup-azure-api-management-instance-ip-address/README.md

```diff
@@ -0,0 +1,49 @@
+# Lookup the IP Address for Azure API Management Instance
+
+## How does it work?
+In certain automated deployment (DevOps) scenarios, for example when you want to whitelist traffic from APIM only, you have the deployed resources (for example Logic Apps) automatically configured with the IP Address of the APIM Instance. This avoids having to manually reconfigure the IP address into all the required resources after deployment.
+
+Luckily, the Azure Resource Manager provides a function that can be used from ARM templates to lookup (or "reference") details from other resources during deployment.
+
+The **`reference()`** function allows you to retrieve the resource in the form of a JSON object and allows you to pick the specific properties that you are interested in:
+```
+reference(resourceId(variables('apimResourceGroup'),'Microsoft.ApiManagement/service/',variables('apimServiceName')),'2019-12-01','Full').properties.publicIPAddresses[0]
+```
+
+A cutdown response JSON object would look similar to the following (many other details omitted for clarity):

```

Comment:
It feels like this is the output of https://github.com/CoditEU/azure-code-snippets/pull/2/files#diff-9d61e9819cea1a99b1516112e2548a6fR10 which is not the case I think?

Maybe we should clarify that so that:
- Folks get to know `reference(resourceId(variables('apimResourceGroup'),'Microsoft.ApiManagement/service/',variables('apimServiceName')),'2019-12-01','Full').properties`
- That function returns JSON
- So if you want to have IP addresses, you can use https://github.com/CoditEU/azure-code-snippets/pull/2/files#diff-9d61e9819cea1a99b1516112e2548a6fR10

What do you think?

Reply:
good catch, thanks!